src/ocean/core/Traits.d

```diff
@@ -2019,7 +2019,7 @@ template ParameterTupleOf( Fn )
         alias Tuple!(Params) ParameterTupleOf;
     else static if( is( Fn Params == delegate ) )
```

Comment:
The change should probably be applied to the other ones as well.

Reply:
I tried, but is ( Fn Params : delegate) issues a compilation error.

Reply:
Maybe nothing is implicitly convertible to a `function` or `delegate` ?

Reply:
Interestingly, it's a parsing error. But the grammar allows it: https://dlang.org/spec/expression.html#is_expression

Reply:
Shouldn't `alias this` of `delegate` member field match this? This looks like different version of the very same bug your linked DMD PR tries to fix.

Not a problem from our side but you may want to check it while you are at it.